docs/source/developers/documentation.rst

```diff
@@ -93,3 +93,20 @@ You can use Docker to build the documentation:
   make -f Makefile.docker docs
 
 The final output is located under ``docs/_build/html``.
+
+.. _building-vcpkg:
+
+Building with Vcpkg
```

Comment:
It seems that this description describes how to install Apache Arrow C++ with vcpkg instead of describing how tobuild with vcpkg.
How about adding this to https://github.com/apache/arrow-site/blob/master/install.md . The file is published as https://arrow.apache.org/install/ .

Reply:
Good suggestion, I will move it to that doc.